src/main/java/com/google/cloud/tools/appengine/api/devserver/RunConfiguration.java

```diff
@@ -81,4 +81,9 @@
   File getDatastorePath();
 
   Map<String, String> getEnvironment();
+
+  /**
+   * Valid only for dev_appserver1.
+   */
+  File getWorkingDirectory();
```

Comment:
Hmm, guess we're already using File everywhere. Probably best to stay consistent.